R/insertPackage.R

```diff
@@ -111,7 +109,11 @@ insertPackage <- function(file,
     }
 
     ## update index
-    write_PACKAGES(pkgdir, type = pkgtype, ...)
+    split_pkgtype <- strsplit(pkgtype,"\\.")[[1L]]
+    write_pkgtype <- 
+        paste(split_pkgtype[seq.int(1L,min(2L,length(split_pkgtype)))],
+              collapse = ".")
```

Comment:
That is excessive indentation which, to me, prohibits readability.  100 columns are fine, most of us have wide-screen monitors.

Reply:
Thanks have to go to Bioconductor's BiocCheck. I didn't even notice it ...